src/features/pages/deriv-ez/hero/data.tsx

```diff
@@ -6,22 +6,26 @@ import DerivEzLogo from 'images/svg/deriv-ez/deriv-ez-logo.svg'
 export const hero_images = {
     hero: (
         <StaticImage
-            src="../../../../images/common/deriv-ez/hero-phone.png"
+            src="../../../../images/common/trading-platforms/deriv-ez/hero.png"
             alt="Deriv EZ online trading platform"
             loading="eager"
             quality={100}
             formats={['avif', 'webp', 'auto']}
+            height={606}
+            width={765}
             objectFit="contain"
             placeholder="none"
```

Comment:
can we use placeholder="blurred" if its possible. Although i know I know its not a scope of your PR. We used this in market banner images and it looks more good. 
Thanks @NikitK-deriv 🙏 
Just a suggestion 😁

Reply:
smart suggestion! as we do it in home page. have sense, much appreciated 🙏